src/main/java/picard/analysis/replicates/CollectIndependentReplicateMetrics.java

```diff
@@ -108,7 +108,7 @@
         summary = "Estimates the rate of independent replication rate of reads within a bam. \n" +
                 "That is, it estimates the fraction of the reads which would be marked as duplicates but " +
                 "are actually biological replicates, independent observations of the data. ",
-        oneLineSummary = "(Experimental) Estimates the rate of independent replication of reads within a bam.",
+        oneLineSummary = "Estimates the rate of independent replication of reads within a bam.",
```

Comment:
This should still be experimental until someone shows that it's good for something...